internal/jobservice/application.go

```diff
@@ -125,8 +125,8 @@ func (a *App) StartUp(ctx context.Context, config *configuration.JobServiceConfi
 			}
 			for _, value := range jobSets {
 				queueJobSet := value.Queue + value.JobSet
-				_, ok := subscribeMap.LoadOrStore(queueJobSet, true)
-				if !ok {
+				_, loaded := subscribeMap.LoadOrStore(queueJobSet, true)
+				if loaded {
```

Comment:
See https://pkg.go.dev/sync#Map.LoadOrStore

We actually only wanted to enter the subscribe logic if the value was *loaded*, otherwise we'd be running the same stream again.

Reply:
but 'loaded' will reflect true if it was already in the map